scripts/generate-dotnet.sh

```diff
@@ -47,8 +47,16 @@ if [ "$PLATFORM" = "ghec" ]; then
 elif [ "$PLATFORM" = "ghes" ]; then
 	NAMESPACE="dotnet-sdk-enterprise-server"
     CSPROJ_PACKAGE_FILE="stage/dotnet/$NAMESPACE/src/GitHub.Octokit.GHES.SDK.csproj"
+    BUILD_YAML="stage/dotnet/$NAMESPACE/.github/workflows/build.yml"
+
+    # Update the PackageId in the .csproj file to include the version
     sed -i "s|<PackageId>GitHub.Octokit.GHES.SDK</PackageId>|<PackageId>GitHub.Octokit.GHES.SDK.$VERSION</PackageId>|" "$CSPROJ_PACKAGE_FILE"
     echo "Updated <PackageId> to GitHub.Octokit.GHES.SDK.$VERSION in $CSPROJ_PACKAGE_FILE"
+
+    # Update the branches node in the publish.yml file to include the current version only
+    # This will set the branches node to an array containing only the new branch name, effectively replacing anything under the node.
+    sed -i'' "s/- main/- $VERSION/" "$BUILD_YAML"
+    echo "Branch replaced with version $VERSION in $BUILD_YAML."
```

Comment:
Can you explain more about what you mean by 

> these changes are intentionally static; the platform version numbers (as branches) are intended to be fixed until we decide otherwise or release a GA where this can be more "fuzzy"

I'm not sure I follow. The versions are defined in [build-dotnet.sh](https://github.com/octokit/source-generator/blob/main/.github/workflows/build-dotnet.yml#L73) matrix, is that what you mean?

Reply:
Hmm this comment no longer applies.